tests/resources/two_hosts_platform.xml

```diff
@@ -2,8 +2,9 @@
 <!DOCTYPE platform SYSTEM "http://simgrid.gforge.inria.fr/simgrid/simgrid.dtd">
 <platform version="4">
   <AS  id="AS0"  routing="Full">
-    <host id="host0" core="4" speed="8095000000f"/>
+    <host id="host0" core="4" speed="8095099999f"/>
```

Comment:
I see the point of having different values, but I'd prefer to have "nice values" that give non-decimal timings in the simulation. Yes, that's picky...